python/ray/util/sgd/v2/tests/test_trainer.py

```diff
@@ -501,6 +501,26 @@ def train():
     assert output == [1, 1]
 
 
+def test_tune_train(ray_start_2_cpus):
+    num_workers = 2
+    epochs = 3
+
+    trainer = Trainer("torch", num_workers=num_workers)
+    MnistTrainable = trainer.to_tune_trainable(fashion_mnist_train_func)
+
+    tune.run(
+        MnistTrainable,
+        num_samples=1,
+        stop={"training_iteration": 2},
```

Comment:
Btw we should get this to work even if there is no stopping condition explicitly passed in.

Reply:
Got rid of the stop condition in this basic test. I can add a separate test case to ensure stop conditions and other tune functionality are working as expected in this integration.